examples/offline/d4rl_cql.py

```diff
@@ -43,6 +43,7 @@ def get_args():
     parser.add_argument("--temperature", type=float, default=1.0)
     parser.add_argument("--cql-weight", type=float, default=1.0)
     parser.add_argument("--with-lagrange", type=bool, default=True)
+    parser.add_argument("--calibrated", type=bool, default=False)
```

Comment:
Does it make sense to change to true? Should also add a description (copied from the docstring)

Reply:
OK!